protected/controllers/ApiController.php

```diff
@@ -52,71 +53,57 @@ public function actionDump()
 
     public function actionDataset()
 	{
-                $status='Published';
-                $id = Yii::app()->request->getParam('id');
-                $doi= Yii::app()->request->getParam('doi');
-                $result= Yii::app()->request->getParam('result');
-                if(!isset($result))
-                {
-                  $result='all';
-                }
-                if(isset($id))
-                {
-                   try{
-                   $model=  Dataset::model()->findByAttributes(array('id'=>$id,'upload_status'=>$status));}
-                   catch(CDbException $e)
-                   {
+        $status = 'Published';
```

Comment:
Shouldn't we add logs? 

Also, can the search really be performed by id?

Reply:
@alli83 what kind of info would you find useful to be logging here?